src/App.jsx

```diff
@@ -7,7 +7,7 @@ import TodoList from './TodoList'
 import './AddTodoForm'
 import AddTodoForm from './AddTodoForm'
 
-const semiPersistentState = () => {
+const usesemiPersistentState = () => {
```

Comment:
we use camel casing so it is easier to read and understand, I will approve the MR but you need to make the changes before you merge

```suggestion
const useSemiPersistentState = () => {
 ````